### ISSUE TITLE
Overfitting on CASIA-v2

### ISSUE THREAD

**Issue by soroushhashemifar** [yelusaleng/RRU-Net#26]
Hi. I tried to finetune your RR-Unet model on CASIA-v2 and CoMoFoD datasets through your repository, but the model overfits. Is this normal?

**Comment by yelusaleng** [comment]
hi, what are your training and testing datasets, respectively?

**Comment by soroushhashemifar** [comment]
Thanks for your response. I have chunked the tampered images of CASIA-v2 into train and val splits with the val ratio of 10%. Around 4000 images for training and 1000 images for validation. Same procedure is applied to CoMoFoD in another experiment. In both cases, it overfits the dataset and the val loss curve starts to increase gradually after 5th epoch. 
My loss function is BCEwithLogits and Adam optimizer is used with LR=1e-3 with LR scheduler.

**Comment by soroushhashemifar** [comment]
I also have an integrated dataset of multiple forgery datasets with more than 16000 tampered images. RR-Unet overfits on this dataset as well.

**Comment by yelusaleng** [comment]
please share the val loss curve on CASIA-2 with me.

**Comment by yelusaleng** [comment]
this figure looks normal, it's not ovefitting or underfitting.

**Comment by soroushhashemifar** [comment]
Hello again. The training on the large dataset is finished and here is the result:

![Training Process for lr-0 001](https://user-images.githubusercontent.com/24815283/189483094-63409b32-2416-4793-9c8b-544f8623952f.png)

The dataset is a mixture of CASIA-v2, IMD2020, and CoMoFod with about 15000 training sample and 1000 validation ones. The resulting model is not able to detect even simple forgeries. Do you have any idea what's going on? (I didn't change your configuration and just changed the dataset)

**Comment by yelusaleng** [comment]
according to the figure, i think the model didn't learn useful information from the mixture dataset.
i want to know how you handle the problem that different size for CASIA, IMD 2020 and COMOFOD.
